### PR TITLE
New pull request

### PR DESCRIPTION
**Pull request by akhil02hero**
I have created few components and yet the style is not done

### CODE REVIEW

File: dashboard/src/index.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+import ReactDOM from 'react-dom';
+import './index.css';
+import Mainbody from './Mainbody';
+import Sidenav from './Sidenav';
+
+
+
+ReactDOM.render(
+  [
+    <Sidenav/>,
```
Comment (fareez-ahamed) [inline dashboard/src/index.js]: Make it render one root component instead of two

File: dashboard/src/Profiles.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+
+const Profiles = ( props) => {
+
+    return (
+        <>
+       <img className="images" src={props.image}/></>
```
Comment (fareez-ahamed) [inline dashboard/src/Profiles.js]: Format your code using prettier

File: dashboard/src/Item1.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import './index.css';
+import RunningCard from './RunningCard';
+const Item1 = () => {
```
Comment (fareez-ahamed) [inline dashboard/src/Item1.js]: Name the components better

File: dashboard/src/Item0.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import './index.css';
+import Child1 from './Child1';
+
+
+const Item0 = () => {
```
Comment (fareez-ahamed) [inline dashboard/src/Item0.js]: Name the components better

File: dashboard/src/Child1.js [hunk 5]
```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import './index.css';
+import logo from './ronaldo.png';
+const Child1 = (props) => {
```
Comment (fareez-ahamed) [inline dashboard/src/Child1.js]: Name the components better

**Review by fareez-ahamed — CHANGES_REQUESTED** [review]
Look at the comments and fix them

File: dashboard/src/index.js (hunk 1 above)
Comment (akhil02hero) [reply dashboard/src/index.js]: I will change it and i will update it

File: dashboard/src/Profiles.js (hunk 2 above)
Comment (akhil02hero) [reply dashboard/src/Profiles.js]: Okay sure

File: dashboard/src/Item1.js (hunk 3 above)
Comment (akhil02hero) [reply dashboard/src/Item1.js]: Okay sure

File: dashboard/src/Child1.js (hunk 5 above)
Comment (akhil02hero) [reply dashboard/src/Child1.js]: Okay sure

File: dashboard/src/Profiles.js (hunk 2 above)
Comment (akhil02hero) [reply dashboard/src/Profiles.js]: Ok sure

File: dashboard/src/Item1.js (hunk 3 above)
Comment (akhil02hero) [reply dashboard/src/Item1.js]: Ok sure

File: dashboard/src/Item0.js (hunk 4 above)
Comment (akhil02hero) [reply dashboard/src/Item0.js]: Ok sure

**Comment by akhil02hero** [comment]
I have renamed all the components

File: dashboard/src/Profiles.js (hunk 2 above)
Comment (fareez-ahamed) [reply dashboard/src/Profiles.js]: Is it  prettified?

File: dashboard/src/Runningprojects.js [hunk 6]
```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import "./index.css";
+import RunningProjectsCard from "./RunningProjectsCard";
+const Runningprojects = () => {
```
Comment (fareez-ahamed) [inline dashboard/src/Runningprojects.js:4]: should be `RunningProjects`

File: dashboard/src/index.js [hunk 7]
```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import App from "./App";
+import "./index.css";
+// import Mainbody from './Mainbody';
+// import Sidenav from './Sidenav';
+
+ReactDOM.render(
+  <>
```
Comment (fareez-ahamed) [inline dashboard/src/index.js:9]: No need to have fragment if there is only one child

**Comment by akhil02hero** [comment]
I made few changes in components

**Review by fareez-ahamed — APPROVED** [review]

**Pull request merged**